fedora-ostree-pruner/fedora-ostree-pruner

```diff
@@ -190,6 +190,11 @@ def prune_prod_repo(test=False):
                                f' repo {OSTREEPRODREPO}: {ref}')
             continue
 
+    # Get the list of aliased refs in the repo
+    cmd = ['ostree', 'refs', '--repo', OSTREEPRODREPO, '--alias']
+    cp = runcmd(cmd, capture_output=True)
+    prod_ref_aliases = [line.split(' ')[0] for line in cp.stdout.decode('utf-8').splitlines()]
```

Comment:
Ouch, parsing non-machine friendly output (and which could theoretically change). Aliases are symlinks, so we could instead do...